opentreemap/treemap/search_fields.py

```diff
@@ -13,33 +13,39 @@
 from treemap.DotDict import DotDict
 from treemap.lib.object_caches import udf_defs
 
-DEFAULT_MOBILE_SEARCH_FIELDS = {
+DEFAULT_MOBILE_SEARCH_FIELDS = DotDict({
     'standard': [
         {'identifier': 'species.id'},
         {'identifier': 'tree.diameter'},
-        {'identifier': 'tree.height'}
+        {'identifier': 'tree.height'},
+        {'identifier': 'plot.owner_orig_id'}
     ],
     'missing': [
         {'identifier': 'species.id'},
         {'identifier': 'tree.diameter'},
-        {'identifier': 'mapFeaturePhoto.id'}
+        {'identifier': 'mapFeaturePhoto.id'},
+        {'identifier': 'plot.owner_orig_id'}
```

Comment:
Mobile needs some more features before searching by Custom ID works well, so we should hold of on making it a default search field.